examples/server_profiles.py

```diff
@@ -187,21 +191,26 @@
         print("No Server Profiles Group found.")
 
 # Get the list of available servers
-print("\nList all available servers associated with a server hardware type and enclosure group")
-available_servers = server_profiles.get_available_servers(
-    enclosureGroupUri=enclosure_group.data["uri"],
-    serverHardwareTypeUri=hardware_type.data["uri"])
-pprint(available_servers)
+# This method works with all the API versions till 1200
+if oneview_client.api_version <= 1200:
+    print("\nList all available servers associated with a server hardware type and enclosure group")
+    available_servers = server_profiles.get_available_servers(
+        enclosureGroupUri=enclosure_group.data["uri"],
+        serverHardwareTypeUri=hardware_type.data["uri"])
+    pprint(available_servers)
 
 # List available storage systems
-print("\nList available storage systems associated with the given enclosure group URI and server hardware type URI")
-available_storage_systems = server_profiles.get_available_storage_systems(
-    count=25, start=0, enclosureGroupUri=enclosure_group.data["uri"],
-    serverHardwareTypeUri=hardware_type.data["uri"])
-pprint(available_storage_systems)
+# This method works with all the API versions till 500
```

Comment:
Mention the method Rest API end points or method name

Reply:
done